### PR TITLE
US123692 - Update functions used by working copy

### PR DESCRIPTION
**Pull request by wongvincent**
https://rally1.rallydev.com/#/29180338367d/iterationstatus?detail=%2Fuserstory%2F478606416460

- Update `checkout` and `checkin` actions to return a siren quiz entity. Temporarily continue supporting `fork` and `merge` actions in these functions.
- Temporarily add a `getCheckoutHref` function while `checkout` is still a GET action.
- Updated timing entity tests with the newly expected serialized timing entity (optimistic loading, both entities will be returned) and the selected radio button will be included as a class.
- Updated quiz entity tests to no longer test `fork` and `merge` actions and update `checkout` to be a `POST` instead of a `GET`.

### CODE REVIEW

File: src/activities/quizzes/QuizEntity.js
Comment (wongvincent) [inline src/activities/quizzes/QuizEntity.js:783]: This is temporary until the `checkout` action is changed from a GET to a POST

File: src/activities/quizzes/QuizEntity.js
Comment (wongvincent) [inline src/activities/quizzes/QuizEntity.js:802]: this `else if` is temporary until `fork` is updated to be `checkout`

File: src/activities/quizzes/QuizEntity.js
Comment (wongvincent) [inline src/activities/quizzes/QuizEntity.js:819]: this `else if` is temporary until `merge` is updated to be `checkin`

**Comment by wongvincent** [comment]
Do not merge until this is merged: https://github.com/Brightspace/lms/pull/6579

File: src/hypermedia-constants.js
Comment (emil-furniss) [inline src/hypermedia-constants.js:369]: Is this right? Looks like the [serializer](https://search.d2l.dev/search?project=lms&full=%22automatic-zero%22&defs=&refs=&path=&hist=&type=&xrd=&si=full&si=full) still uses `automatic-zero`?

**Review by emil-furniss — APPROVED** [review]

File: src/hypermedia-constants.js
Comment (wongvincent) [reply src/hypermedia-constants.js:369]: Good question. I'm updating it here! https://github.com/Brightspace/lms/pull/6616/files#diff-3ac4b97bf1fa5b1a1c7096d96a97e57c4a038805fd97ddbd50b790636a1a3426R332

**Pull request merged**
